env2secret.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
I agree on dropping the makefile and keeping only Helm. Both our teams are using Helm so lets keep this simple.